### ISSUE TITLE
Error when serializing an Instant

### ISSUE THREAD

**Issue by minborg** [OpenHFT/Chronicle-Wire#341]
Reproduce:

```
    @Test
    public void instant() {
        final MyClass source = new MyClass();
        source.instant = Instant.ofEpochMilli(1_000_000_000_000L);
        System.out.println("source.instant = " + source.instant);

        final Bytes<?> bytes = Bytes.allocateElasticOnHeap(64);
        final Wire wire = WireType.BINARY.apply(bytes);

        wire.getValueOut().object(source);
        final MyClass target = wire.getValueIn().object(source.getClass());
        Assert.assertEquals(source, target);

    }

    static final class MyClass extends SelfDescribingMarshallable {

        Instant instant;

        @Override
        public boolean equals(Object o) {
...

```

This is the root cause for https://github.com/OpenHFT/Chronicle-Queue/issues/945

**Comment by peter-lawrey** [comment]
Currently there lacks a means of ensuring there is a one to one match in how a type is serialized for built in classes, without a relevant unit test.

**Comment by hft-team-city** [comment]
Released in [Chronicle-Wire-2.22ea11](https://github.com/OpenHFT/Chronicle-Wire/releases/chronicle-wire-2.22ea11), [BOM-2.22ea69](https://github.com/OpenHFT/OpenHFT/releases/chronicle-bom-2.22ea69)
